### ISSUE TITLE
startDebug command is deprecated

### ISSUE THREAD

**Issue by WebFreak001** [Pure-D/code-d#145]
https://github.com/Pure-D/code-d/blob/master/src/compile-buttons.ts#L164

Should be replaced with `vscode.debug.startDebugging`

**Comment by Barugon** [comment]
Any estimate on when this might be implemented?

**Comment by WebFreak001** [comment]
Hm I don't really want to add this to code-d ~master (the normal version on the marketplace) because the serve-d version is the new better version which should be worked on and it's just duplicating work doing these fixes to both branches. Right now I am not really motivated to work on this so much though, because every little change to workspace-d or serve-d will require a new release with multiple precompiled binaries once I released it. Also my virtualbox setup is broken right now so I couldn't provide windows binaries anyway. So that's why I don't really want to publish the serve-d branch yet. Maybe if I had some automated binary build solution this would be easier, but last time I tried cross compiling to windows from linux using LDC I didn't manage to do so. I could theoretically setup a build server on webfreak.org if someone could show me how to cross compile properly with LDC. I could then also provide builds for DCD, etc. because they are often a problem for windows users installing my extension because of weird windows path restrictions.

tldr: give me 5 minutes, but it will only be on the serve-d branch

**Comment by Barugon** [comment]
I'm using the beta version from the marketplace, which I thought was using serve-d. Anyway, both versions in the marketplace give the same error when attempting to launch the debugger. Also, I'm on Linux if that makes any difference.

**Comment by Barugon** [comment]
Update from the marketplace seems to be working now. Thanks!  :)
